.github/workflows/ci.yml

```diff
@@ -0,0 +1,70 @@
+name: Backend CI
+
+on:
+  push:
+    branches-ignore:
+      - 'master'
+    tags-ignore:
+      - '**'
+    paths:
+      - '**.php'
+      - '**.yml'
+      - '**.yaml'
+      - 'app/schema/**.xml'
+      - 'composer.lock'
+      - 'phpcs.xml.dist'
+      - 'phpunit.xml.dist'
+      - 'psalm.xml'
+      - 'psalm-baseline.xml'
+
+jobs:
+  test:
+    name: Test
```

Comment:
```suggestion
    name: Test (PHP ${{ matrix.php-version }})
```

Reply:
I don't think you have access to the matrix at this point.

Reply:
So this is not going to work either? https://github.com/MyOnlineStore/common-factories/blob/8e07275aacb360e94ff64248c476504a4a233498/.github/workflows/ci.yml#L7

Reply:
Seems to be working :shrug:

Reply:
I don't quite understand the magic behind the title of the job name, in the monolith we also test with a matrix but the `test.name` is just "Test", yet the version is behind it https://github.com/MyOnlineStore/myonlinestore/blob/02fc69bc41a89ef05fad56ac68ee7f22bcad8c96/.github/workflows/backend-ci.yml#L22

Reply:
There's a slight difference in outcome, `Test (7.2)` vs `Test( PHP 7.2)`
You're suggestion is a bit more explicit, but I don't have a strong preference on which way we go.
https://github.com/MyOnlineStore/common-factories/actions/runs/668325245
https://github.com/MyOnlineStore/myonlinestore/actions/runs/1097418223